### DISCUSSION TITLE
Meeting Minutes template

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#473] [Meetings]
To document the minutes for a meeting, copy the markdown template below into a new discussion item and fill it in!

-------------------------
```
# Attendees
- @jwildfire 
- @xni7 

# Agenda/Notes

1. Agenda item #1
2. Agenda item #2 

# Action Items

- AI1
- AI2

```

**Comment by mli1** [comment]
Jeremy, thank you for _providing_ the **template**.
